### PR TITLE
Learners coaches

### PR DESCRIPTION
**Pull request by indirectlylit**
### Summary

Addresses comment in #3400:

>  it might aid discovery of the action buttons if they were contextualized to their sections -- so, "Assign coaches" button next to the "Coaches" section, and "Enroll learners" next to the "Learners" section

| Old | New |
|--|--|
| image |  image |

### Reviewer guidance

look good? work right?

### References

https://github.com/learningequality/kolibri/issues/3400

----

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

File: kolibri/plugins/facility_management/assets/src/views/class-edit-page/index.vue [hunk 1]
```diff
@@ -173,48 +179,24 @@
...
-  .header
-    font-weight: normal
+  .top-margin
+    margin-top: 24px
 
-  .user-roster
-    overflow-x: auto
-    overflow-y: hidden
+  // overwrite global styling
+  // TODO - find a better way of doing this
```
Comment (christianmemije) [inline kolibri/plugins/facility_management/assets/src/views/class-edit-page/index.vue:198]: I guess we could give the k-grid a prop that removes padding?

Comment (indirectlylit) [reply kolibri/plugins/facility_management/assets/src/views/class-edit-page/index.vue:198]: really ought to be getting rid of these, and replacing them with something that doesn't introduce left- and right- padding to the outer elements. The goal was just gutters - not outer margins

Comment (indirectlylit) [reply kolibri/plugins/facility_management/assets/src/views/class-edit-page/index.vue:198]: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/styles/main.styl#L118

**Comment by khangmach** [comment]
Can we use the standard flat secondary for the remove buttons? Example seen here:
screen shot 2018-04-18 at 12 44 11 pm

**Comment by indirectlylit** [comment]
> Can we apply the standard flat secondary for the remove buttons? Example seen here:

Yeah, good idea. Will open a new PR

**Pull request merged**